src/MOCKernel.cpp

```diff
@@ -0,0 +1,165 @@
+#include "MOCKernel.h"
+#include "TrackGenerator.h"
+
+/**
+ * @brief Constructor for the MOCKernel assigns default values
```

Comment:
The text "assigns default values" in the `@brief` comment block isn't very informative unless you expand upon this in a `@details` block (*i.e.*, a maximum optical path length is assigned).